test/e2e/config/docker.yaml

```diff
@@ -401,6 +401,7 @@ variables:
   CLUSTER_TOPOLOGY: "true"
   EXP_RUNTIME_SDK: "true"
   EXP_MACHINE_SET_PREFLIGHT_CHECKS: "true"
+  EXP_PRIORITY_QUEUE: "false"
```

Comment:
Don't we want to enable it for testing?
Or at least in some test?

Reply:
Not sure how to enable this only for some tests.

I think we have to preserve our current test coverage for the feature gate being off as that is what (almost?) everyone will run in production.

I personally would mainly use this feature gate for now to enable the feature in scale tests (outside of CI)

Reply:
+1 to experiment outside CI first